README.md

```diff
@@ -1 +1,308 @@
-# measurement-opt
\ No newline at end of file
+# Measurement This, Not That: Optimizing the Cost and Model-Based Information Content of Measurements
+
+Authors: Jialu Wang, Zedong Peng, Ryan Hughes, Debangsu Bhattacharyya, David E. Bernal Neira, Alexander W. Dowling 
+
+This repository contains code and results for paper: Measure this, not that: Optimizing the cost and model-based information content of measurements 
+
+## Installation instructions 
+
+The following instructions assume you have anaconda installed. We suggest create an environment with the following commands to run code: 
+
+### Step 1: create a new environment 
+- create new environment, called for e.g. `measurement_optimization`, with `conda` with `Python` version 3.8
+
+`conda create --name measurement_optimization python=3.8`
+
+`conda activate measurement_optimization`
+   
+### Step 2: install `IDAES-PSE`
+- this step provides `Ipopt` solver, but this solver is not necessary for reproducing paper results
+- we did not test if the code can run without this step
+
+`pip install idaes-pse` 
+
+`idaes get-extensions`
+   
+### Step 3: install `Pyomo` from specified branches
+- install from the following branch for a generalization version of Mindtpy:
+
+`pip install git+https://github.com/ZedongPeng/pyomo.git@add_mindtpy_callback`
+
+   
+### Step 4: install `GurobiPy`
+- this is needed only for solving mixed-integer problems
+
+  `conda install -c gurobi gurobi`
+   
+### Step 5: install `CyIpopt`
+- this is needed only for D-optimality problems with grey-box modules
+
+   `conda install -c conda-forge cyipopt`
+
+### Software versions we use for the results 
+
+`Python`: 3.8
+
+`IDAES-PSE`: 2.2.0
```

Comment:
Verify this is not needed. I suspect once you try this in a new environment without IDAES installed, you'll find we need to install other dependencies such as Jupyter, matplotlib, etc.

Reply:
yes. Updated.